src/coreComponents/physicsSolvers/contact/SolidMechanicsEmbeddedFractures.cpp

```diff
@@ -549,14 +549,16 @@ void SolidMechanicsEmbeddedFractures::applyTractionBC( real64 const time_n,
   } );
 }
 
-real64 SolidMechanicsEmbeddedFractures::calculateResidualNorm( DomainPartition const & domain,
+real64 SolidMechanicsEmbeddedFractures::calculateResidualNorm( real64 const & time,
```

Comment:
when you merge, let us create an issue to remind us to change the contact solvers too.